scipy/stats/_continuous_distns.py

```diff
@@ -4346,8 +4346,19 @@ def _stats(self, a, moments='mvsk'):
         return m1, m2, g1, g2
 
     def _entropy(self, a):
-        return a - (a+1.0) * sc.psi(a) + sc.gammaln(a)
-
+        def regular(a):
+            h = a - (a + 1.0) * sc.psi(a) + sc.gammaln(a)
+            return h
+
+        def asymptotic(a):
+            # gammaln(a) ~ a * ln(a) - a - 0.5 * ln(a) + 0.5 * ln(2 * pi)
+            # psi(a) ~ ln(a) - 1 / (2 * a)
+            h = ((1 - 3*np.log(a) + np.log(2) + np.log(np.pi))/2
+                 + 2/3*a**-1 + a**-2/12 - a**-3/90 - a**-4/120)
+            return h
+
+        h = _lazywhere(a >= 2e2, (a,), f=asymptotic, f2=regular)
+        return h
```

Comment:
```suggestion
        return h

```
PEP8 but let's NOT merge this one if CI doesn't complain. Not worth another run. Please fix it in another PR.

Reply:
CI won't complain, the linter's broken (unless it's been fixed very recently.)

Reply:
If it's broken, that must be very recent. IIUC, we switched linters about a month ago and it not currently checking all the rules we used to.

Reply:
Then let's not worry about the PR until the linter is fully up to snuff. Also, a PR just for a single pep8 violation seems a bit wasteful. It feels like it's be better to try and catch all the violations in one big PR once the linter can catch them all.

Reply:
I mean in any one of the other PRs you have open.